### PR TITLE
Add --auto-approve to `db up`

### PR DESCRIPTION
**Pull request by rockymeza**
If you try to deploy a change that has a migration that will have data loss, the prisma migrate command will ask the user for confirmation. This is great for local development, but not great for continuous deployment. On Netlify this will cause deployments to just hang:

image

This PR adds an `--auto-approve` option to `yarn rw db up` that just applies `--auto-approve` to the `prisma migrate` command.

I also think we might want to add this `--auto-approve` option to the `netlify.toml` build command.

### CODE REVIEW

**Comment by rockymeza** [comment]
In the meantime, here is a workaround that I've been using

```diff
diff --git a/netlify.toml b/netlify.toml
index 92b4c12..05a39df 100644
--- a/netlify.toml
+++ b/netlify.toml
@@ -1,5 +1,5 @@
 [build]
-  command = "yarn rw db up --no-db-client && yarn rw build"
+  command = "cd api && yarn prisma migrate up --experimental --auto-approve --create-db && cd .. && yarn rw build"
   publish = "web/dist"
   functions = "api/dist/functions"
```

File: packages/cli/src/commands/dbCommands/up.js [hunk 1]
```diff
@@ -17,6 +17,11 @@ export const builder = (yargs) => {
       description: 'Generate the Prisma client',
       type: 'boolean',
     })
+    .option('autoApprove', {
+      default: false,
+      description: 'Skip interactive approval before migrating',
+      type: 'boolean',
+    })
```
Comment (jtoar) [inline packages/cli/src/commands/dbCommands/up.js:24]: This looks great!

**Comment by peterp** [comment]
This is great, I'm going to add this as the default to the `netlify.toml`.

I also think it makes a lot of sense to allow arguments (that we don't specify) to be forwarded over to the prisma cli, if we do that then we don't have to add any new code to support things like this.

**Comment by jtoar** [comment]
One thing to note if we go that route is that the help message will just have to output a link to the prisma docs to be useful, since yargs uses the options explicitly specified in `builder` to construct the help message.

Maybe we could "override" help by making it a bona fide argument that just triggers the prisma help output? Then the help output of our commands wouldn't look uniform, but that might not matter that much.

**Pull request merged**